x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/select_rule_type/index.tsx

```diff
@@ -47,11 +90,10 @@ export const SelectRuleType: React.FC<SelectRuleTypeProps> = ({ field, isReadOnl
         <EuiFlexItem>
           <EuiCard
             title={i18n.ML_TYPE_TITLE}
-            description={license ? i18n.ML_TYPE_DESCRIPTION : i18n.ML_TYPE_DISABLED_DESCRIPTION}
-            isDisabled={!license}
+            description={<MlCardDescription hasValidLicense={hasValidLicense} />}
             icon={<EuiIcon size="l" type="machineLearningApp" />}
```

Comment:
I played with adding:
```
isDisabled={isReadOnly || !hasValidLicense}
```

To see what it looked like if the whole thing was diasbled.

What you have now:
<img width="263" alt="Screen Shot 2020-03-20 at 4 59 43 PM" src="https://user-images.githubusercontent.com/1151048/77212475-ed941900-6acc-11ea-899e-1a0b4a1fc05c.png">

vs if the whole thing was disabled:
<img width="252" alt="Screen Shot 2020-03-20 at 5 03 55 PM" src="https://user-images.githubusercontent.com/1151048/77212484-f4bb2700-6acc-11ea-9045-6f2dddf38c92.png">

Up to you and design on which is better but I prefer the whole thing being disabled.

Reply:
I ended up going with the full-disabling, as it is more visually striking and a better CTA